File: MyProfile/Models/Work.cs
using System;

namespace MyProfile.Models
{
    public class Work
    {
        public int Id { get; set; }
        public string Company { get; set; }
        public string Position { get; set; }
        public string Website { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Summary { get; set; }
    }
}

File: MyProfile/client/src/components/work/Work.js
import { Link } from "react-router-dom"
import { Card, CardBody } from "reactstrap"
import './work.css'

export const Work = ({work}) => {
    return (
        <>
        <div className="work">
        <Card
        color="secondary"
        >
            <CardBody>
                <h1>{work.company}</h1>
                <p>{work.position}</p>
                <p className="link">
                    <a href= {work.website} target="_blank">View Website</a>
                </p>
                <p>{work.summary}</p>
                <footer>Start Date: {work.startDate}</footer>
                <footer>End Date: {work.endDate}</footer>
            </CardBody>
        </Card>
        <br/>
        </div>
        </>
    )
}